src/components/authentication/SignUpFields.tsx

```diff
@@ -1,21 +1,38 @@
 import React from "react";
-import { TextField, makeStyles } from "@material-ui/core";
+import {
+  TextField,
+  Typography,
+  makeStyles,
+  RadioGroup,
+  FormControlLabel,
+  Radio
+} from "@material-ui/core";
+import { SignupInputDataType } from "../../utils/signup/signupUtils";
+import CountrySelect from "../user/CountrySelect";
 
 type Props = {
+  inputData: SignupInputDataType;
   onChangeData: (field: string, e: any) => void;
 };
 
-const GAP_BETWEEN_INPUTS = 10;
-
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   signupFields__BtnContainer: {
-    marginTop: GAP_BETWEEN_INPUTS,
-    marginBottom: GAP_BETWEEN_INPUTS
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1)
+  },
+  signupFields__TextSeparator: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1.25)
   }
-});
+}));
 
 function SignUpFields(props: Props) {
-  const { onChangeData } = props;
+  const { onChangeData, inputData } = props;
+  const { password, confirmPassword } = inputData;
+  const confirmPasswordHelperText =
+    confirmPassword && password !== confirmPassword
+      ? "Password must match"
+      : "";
   const styles = useStyles();
   return (
     <>
```

Comment:
I am thinking here we coul also define all fields in an array and map on this array to return DOM elements - will remove a lot of code duplication. But not compulsory of course